hashmap.js

```diff
@@ -85,8 +85,8 @@
 		type:function(key) {
 			var str = Object.prototype.toString.call(key);
 			var type = str.slice(8, -1).toLowerCase();
-			// Some browsers yield DOMWindow for null and undefined, works fine on Node
-			if (type === 'domwindow' && !key) {
+			// Some browsers yield DOMWindow or Window for null and undefined, works fine on Node
+			if (!key && type === 'domwindow' || type === 'window') {

```

Comment:
The if is missing parens, it should be !key && (type === 'domwindow' || type === 'window') but it could reduced to !key && type.slice(-6) === 'window', I prefer the latter but either would be fine.